qemu/tests/cfg/block_boot_under_low_speed.cfg

```diff
@@ -0,0 +1,17 @@
+- block_boot_under_low_speed:
+    only Windows
+    type = block_boot_under_low_speed
+    login_timeout = 2000
+    start_vm = no
+    nfs_exports =/home/tmp/exports
+    nfs_path = /home/tmp/nfs
+    host_script = block_device/speed.sh
+    pre_command = "if ! rpm -qa|grep nfs-utils;then yum install -y nfs-utils; fi;"
+    pre_command += "mkdir -p ${nfs_exports} ${nfs_path};"
+    pre_command += echo "${nfs_exports} *(rw,no_root_squash)" >> /etc/exports;cat /etc/exports;
+    pre_command += "if systemctl is-active nfs-server; then systemctl reload nfs-server;"
+    pre_command += "else systemctl start nfs-server;fi &&"
+    pre_command += "mount 127.0.0.1:${nfs_exports} ${nfs_path} "
+    post_command = "umount -lf ${nfs_path};"
+    post_command += "sed -i -e '/\/home\/tmp\/exports/d' /etc/exports;cat /etc/exports;"
+    post_command += "rm -rf ${nfs_exports} ${nfs_path};"
```

Comment:
I think we can have a try to call the module `virttest.nfs.py` instead of `pre_command` and `post_command`.